### PR TITLE
Added Getting Started section to README

### PR DESCRIPTION
**Pull request by danlope**
Added Getting Started to guide any user to get started using the Webhint Extension in their own project

### CODE REVIEW

**Comment by molant** [comment]
@danlope I'm looking into this right now. I might push some commits tto align with the formatting we use in the rest of the project.

Thanks for the PR!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,53 @@
...
+- Marketplace will not allow size greater than 1000 files (or assets)
+- Remove node modules from vss-extension.json "files" attribute
+- Update version number in vss-extension.json to a greater version (required) 
+- Run the following command to create extension and upload new published file without node modules to marketplace (should allow)
+    > tfx extension create
+- Add node modules back to vss-extension.json
+- Update version number again in vss-extension.json to a greater version (required every update) 
+- Recreate extension and update file in marketplace. 
+    > tfx extension create 
+- NOTE: An update to the file in Marketplace will allow size greater than 1000 files. An initial upload will not allow such large sizes.
```
Comment (molant) [inline README.md]: If I've understood this correctly we only need to publish without `node_modules` once, correct?

Also if we follow this steps the extension will published into the marketplace and available to any organization, correct?

Comment (danlope) [reply README.md]: That is correct. The initial load limits the file size so removing the node modules will allow for the upload. After upload, node modules can be added back to the project and version number of extension must be increased to re-create. Once the extension is re-created you can update the uploaded folder. No size limit with an update.  

The extension will be published and available for any org once these steps are completed.

Comment (molant) [reply README.md]: Perfect, thanks!

**Pull request closed without merging**